src/common/internal_types.cpp

```diff
@@ -0,0 +1,23 @@
+//===----------------------------------------------------------------------===//
+//
+//                         Peloton
+//
+// internal_types.cpp
+//
+// Identification: src/common/internal_types.cpp
+//
+// Copyright (c) 2015-2018, Carnegie Mellon University Database Group
+//
+//===----------------------------------------------------------------------===//
+
+#include "common/internal_types.h"
+
+namespace terrier {
+
+// FileHandle INVALID_FILE_HANDLE;
+//
+
+// For threads
+size_t CONNECTION_THREAD_COUNT = 1;
```

Comment:
Maybe move into 'common/constants.h' or start a `network/network_defs.h` as is customary in the new code base,